src/main/java/com/iexec/common/replicate/ReplicateActionResponse.java

```diff
@@ -0,0 +1,44 @@
+package com.iexec.common.replicate;
+
+import com.iexec.common.chain.ChainReceipt;
+import com.iexec.common.replicate.ReplicateStatus;
+import com.iexec.common.replicate.ReplicateStatusCause;
+import com.iexec.common.replicate.ReplicateStatusDetails;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ReplicateActionResponse {
+
+    private ReplicateStatus status;
```

Comment:
Don't we want a boolean? (Sorrry)

Reply:
No we don't need it since we'll do the same thing independently from whether it is a fail or a success.